Sources/PubNub/PubNub.swift

```diff
@@ -38,13 +38,18 @@ public class PubNub {
   ///   - session: Session used for performing request/response REST calls
   ///   - subscribeSession: The network session used for Subscription only
   ///   - fileSession: The network session used for File uploading/downloading only
+  ///   - logger: The logger to be used for the PubNub SDK
   public convenience init(
     configuration: PubNubConfiguration,
     session: SessionReplaceable? = nil,
     subscribeSession: SessionReplaceable? = nil,
-    fileSession: URLSessionReplaceable? = nil
+    fileSession: URLSessionReplaceable? = nil,
+    logger: PubNubLogger = PubNubLogger.defaultLogger()
```

Comment:
With this in place it looks like there is no way to init PubNub instance quietly without logger at all. 
In Objective-C and JavaScript SDKs we have a way to set desired log level and list of custom loggers.

Reply:
I can change this behavior, so the default logger value produced by `PubNubLogger.defaultLogger()` could be:

`PubNubLogger(levels: .none)`

This will disable all the logging. I can also make `logger` variable as optional and nil by default as an alternative approach